src/content/docs/fr/guides/configuring-astro.mdx

```diff
@@ -1,169 +1,119 @@
+De plus, les différents hôtes de déploiement peuvent avoir un comportement différent en ce qui concerne les barres obliques à la fin de vos URLs. (par exemple, `example.com/about` vs `example.com/about/`). Une fois que votre site est déployé, vous pouvez avoir besoin de configurer votre préférence [`trailingSlash`](/fr/reference/configuration-reference/#trailingslash).
 
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config'
-
-export default defineConfig({
-  // Se dirige vers le répertoire "./foo" dans le répertoire de travail actuel
-  root: 'foo',
-  // Se dirige vers le répertoire "./foo/public" dans le répertoire de travail actuel
-  publicDir: 'public',
-})
-```
-
-Pour référencer un fichier ou un répertoire relatif au fichier de configuration, utilisez `import.meta.url` (sauf si vous écrivez un fichier common.js `astro.config.cjs`).
-
-```js "import.meta.url"
-// astro.config.mjs
+```js title="astro.config.mjs"
 import { defineConfig } from 'astro/config'
 
 export default defineConfig({
-  // Se dirige vers le répertoire "./foo", relatif à ce fichier de configuration
-  root: new URL("./foo", import.meta.url).toString(),
-  // Se dirige vers le répertoire "./public", relatif à ce fichier de configuration
-  publicDir: new URL("./public", import.meta.url).toString(),
+  site: 'https://www.example.com',
+  base: '/docs',
+  trailingSlash: 'always',
 })
 ```
+### Ajouter les métadonnées du site
 
-:::note
-Les propriétés `import.meta` spécifiques à Vite, comme `import.meta.env` ou `import.meta.glob`, ne sont _pas_ accessibles depuis votre fichier de configuration. Nous recommandons des alternatives comme [dotenv](https://github.com/motdotla/dotenv) ou [fast-glob](https://github.com/mrmlnc/fast-glob) pour ces cas d'utilisation respectifs. En outre, [les alias de chemin tsconfig](https://www.typescriptlang.org/tsconfig#paths) ne seront pas résolus. Utilisez des chemins relatifs pour les importations de modules dans ce fichier.
-:::
-
-## Personnalisation des noms de fichiers de sortie
+Astro n'utilise pas son fichier de configuration pour les données de référencement ou les métadonnées, mais seulement pour les informations nécessaires à la construction du code de votre projet et à son rendu en HTML.
 
-Pour le code qu'Astro traite, comme les fichiers JavaScript ou CSS importés, vous pouvez personnaliser les noms de fichiers de sortie en utilisant [`entryFileNames` (EN)](https://rollupjs.org/configuration-options/#output-entryfilenames), [`chunkFileNames` (EN)](https://rollupjs.org/configuration-options/#output-chunkfilenames), et [`assetFileNames` (EN)](https://rollupjs.org/configuration-options/#output-assetfilenames) dans une entrée `vite.build.rollupOptions` de votre fichier  `astro.config.*`.
+Au lieu de cela, ces informations sont ajoutées à votre page `<head>` dans les balises HTML `<link>` et `<meta>`, comme si vous écriviez des pages HTML simples.
```

Comment:
I would tend to use `balise` in front of `<head>`, but that might be a lot of `balises`... maybe that's enough:
```suggestion
Au lieu de cela, ces informations sont ajoutées au `<head>` de votre page dans les balises HTML `<link>` et `<meta>`, comme si vous écriviez des pages HTML simples.
```